Realm.podspec

```diff
@@ -30,7 +30,8 @@ Pod::Spec.new do |s|
                               'include/Realm/RLMRealm.h',
                               'include/Realm/RLMResults.h',
                               'include/Realm/RLMSchema.h',
-                              'include/Realm/Realm.h'
+                              'include/Realm/Realm.h',
+			      'include/Realm/RLMRealm_Dynamic.h'
```

Comment:
I don't think so because would rather not force dev to adjust their install of Realm to use a component...